proposals/proposal_process.md

```diff
@@ -0,0 +1,183 @@
+# WECG Proposal Process
+
+## Summary
+
+This document outlines a new proposal process for making changes to public
+extension APIs.  This process should be used for all future changes to public
+extension APIs.
+
+## Background and Motivation
+
+Historically, different browsers had (very) different forms of "web extension"
+(also known as "browser extension") platforms, including Chrome Extensions and
+Firefox Add-Ons.  Over the past several years, browser vendors have aligned on
+a common set of WebExtension APIs and all major browser vendors (including
+Google Chrome, Mozilla Firefox, and Apple Safari, along with the majority of
+other Chromium-based browsers such as Microsoft Edge, Opera, Brave, and more)
+support the same general API and behavior.
+
+With this convergence on a common platform, browser vendors and developers have
+been working to ensure better cross-browser alignment on new and existing APIs,
+along with future API changes.  This is the primary purpose of the
+[WebExtensions Community
+Group](https://github.com/w3c/webextensions/blob/main/charter.md), or WECG.
+
+However, due to the previous history and implementation practices in which
+every browser had complete autonomy over their own API design and
+implementation, not all APIs and API changes have been brought to the WECG for
+discussion.  At best, this results in a missed opportunity for broader
+community discussion and creates more time between when browsers can support a
+similar API; at worst, this results in significant platform fragmentation, to
+the detriment of the ecosystem.
+
+When changes and proposals have been brought to the WECG, they are frequently
+presented in different formats and manners based on historical precedence for a
+specific browser.  This makes it more difficult to review, as well as risking
+the omission of important criteria or details.
+
+In order to address these concerns, we propose a defined and consistent
```

Comment:
```suggestion
In order to address these concerns, we use a defined and consistent
```

"propose ... a proposal process" seems indefinite. The intent is for this to be the process to use, so let's replace "propose" with a more definite verb, e.g. "use" or "apply".